public/data/Minecodes.json

```diff
@@ -0,0 +1,80 @@
+{
+    "name": "Minecodes",
+    "bio": "Hobby coder that builds apps for Discord, Markdown, GitHub, Linux, Windows, Guilded and Minecraft",
+    "avatar": "https://github.com/Minecodes.png",
+    "links": [
+        {
+            "name": "Visit my Youtube Channel",
+            "url": "https://youtube.com/Minecodes",
+            "icon": "youtube"
+        },
+        {
+            "name": "Visit Website",
+            "url": "https://minecodes.de",
+            "icon": "globe"
+        },
+        {
+            "name": "Read my Blog",
+            "url": "https://dev.to/minecodes",
+            "icon": "book"
+        },
+        {
+            "name": "Contact us via mail",
+            "url": "mailto:minecodes13@gmail.com",
+            "icon": "envelope"
+        },
+        {
+            "name": "Visit GitHub",
+            "url": "https://github.com/Minecodes",
+            "icon": "github"
+        },
+        {
+            "name": "Join our Discord Community",
+            "url": "https://discord.minecodes.de",
+            "icon": "discord"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/Minecodes1",
+            "icon": "twitter"
+        }
+    ],
+    "milestones": [
+        {
+            "title": "Started YouTube",
+            "image": "https://github.com/Minecodes.png",
+            "date": "April 2021",
+            "icon": "youtube",
+            "color": "red",
+            "description": "First video",
+            "url": "https://youtu.be/RpjnTpVx6tg"
+        },
+        {
+          "title": "First Github project",
+          "image": "https://raw.githubusercontent.com/Minecodes/randomfox/master/assets/randomfox.png",
+          "date": "Januar 2021",
+          "icon": "github",
+          "color": "orange",
+          "description": "Created randomfox",
+          "url": "https://github.com/Minecodes/randomfox"
+        },
+        {
+            "title": "41 sups on Youtube",
+            "image": "https://i9.ytimg.com/vi_webp/TEH1sveCv2g/mqdefault.webp?v=60ed1d46&sqp=CLzFk40G&rs=AOn4CLBufwFB5eIyC8s5KxLdYPZD_pgq5g",
+            "date": "July 2021",
+            "icon": "youtube",
+            "color": "red",
+            "description": "I got 41 sups there and not a week later 45 sups",
```

Comment:
subs

Reply:
Thanks! I am not so good in English at the moment, because I am a German student.